documentation/guidance/plan-the-process.md

```diff
@@ -141,12 +141,12 @@ You can filter your results by grant size, date, location, grant programme, fund
 
 For more guidance about how to explore 360Giving data in GrantNav visit the <a href="https://help.grantnav.threesixtygiving.org/en/latest" target="_blank">GrantNav Help site.</a>
 
-#### 360Insights
-<a href="https://insights.threesixtygiving.org" target="_blank">360Insights</a> is a tool to help you understand funders better. You can combine and visualise 360Giving and charity data, and explore funders across different areas – from their grant dates to types of recipients.
+#### GrantVis
+<a href="https://grantvis.threesixtygiving.org" target="_blank">GrantVis</a> is a tool to help you understand funders better. You can combine and visualise 360Giving and charity data, and explore funders across different areas – from their grant dates to types of recipients.
 
 You can filter the 360Insights dashboard to see results for grants based on grant programme, organisation type, size or region.
 
-![Screenshot of 360Insights](../../assets/Example_360Insights_view.png)
+![Screenshot of GrantVis, formerly known as 360Insights](../../assets/Example_360Insights_view.png)
```

Comment:
This screenshot can be replaced with the updated landing page 
![Screenshot 2025-03-04 at 14 56 02](https://github.com/user-attachments/assets/46279dd7-1e51-4dff-a025-d3cf0d9dc672)

Reply:
```suggestion
![Screenshot of GrantVis, formerly known as 360Insights](../../assets/Example_GrantVis_view.png)
```

Reply:
@KDuerden Checking it is ok to update the image from Insights to GrantVis

Reply:
yes, thank you for doing this - that's much better.